ipatests/test_integration/test_commands.py

```diff
@@ -130,6 +131,27 @@ def get_cert_base64(self, host, path):
         cader = ssl.PEM_cert_to_DER_cert(cacrt)
         return base64.b64encode(cader).decode('ascii')
 
+    def test_aes_sha_kerberos_enctypes(self):
+        """Test AES SHA 256 and 384 Kerberos enctypes enabled
+
+        AES SHA 256 and 384-bit enctypes supported by MIT kerberos but
+        was not enabled in IPA. This test is to check if these types are
+        enabled.
+
+        related: https://pagure.io/freeipa/issue/8110
+        """
+        tasks.kinit_admin(self.master)
+        realm = self.master.domain.realm
+        suffix = realm_to_suffix(realm)
+        dn = DN(("cn", realm), ("cn", "kerberos")) + suffix
```

Comment:
IMHO the extra variables aren't needed and you don't need to add the DNs:

dn = DN(("cn", realm), ("cn", "kerberos"), realm_to_suffix(self.master.domain.realm))